build-scripts/config_common.cmake

```diff
@@ -287,3 +287,7 @@ if (WAMR_BUILD_STACK_GUARD_SIZE GREATER 0)
     add_definitions (-DWASM_STACK_GUARD_SIZE=${WAMR_BUILD_STACK_GUARD_SIZE})
     message ("     Custom stack guard size: " ${WAMR_BUILD_STACK_GUARD_SIZE})
 endif ()
+if (WAMR_SGX_IPFS EQUAL 1)
+    add_definitions (-DWAMR_SGX_IPFS=1)
```

Comment:
Had better use `WASM_ENABLE_SGX_IPFS` to align with other macros.
And add lines in `core/config.h`:
```C
#ifndef WASM_ENABLE_SGX_IPFS
#define WASM_ENABLE_SGX_IPFS 0
#endif
```